packages/css/src/components/whitebox/whitebox.template.ts

```diff
@@ -0,0 +1,44 @@
+import { Whitebox } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map';
+
+import { iconTemplate } from '../icon/icon.template';
+import { imageTemplate } from '../image/image.template';
+
+export function whiteboxTemplate({ count, icon, iconText, image, label, richContent, title }: Whitebox<TemplateResult>) {
+  return html`
+    <div class="dso-whitebox ${classMap({ 'dso-has-counter': !!(count || (icon && iconText)) })}">
+      ${icon && iconText
+        ? html`
+          <div class="dso-step-counter">
+            ${iconTemplate({ icon: 'check' })}
+            <span class="sr-only">${iconText}</span>
+          </div>
+        `
+        : nothing
+      }
+      ${count
+        ? html`
+          <div class="dso-step-counter">
+            ${count}
+          </div>
+        `
+        : nothing
+      }
+      <div class="dso-whitebox-title">
+        <h2>${title}</h2>
+      </div>
+      <div class="dso-whitebox-link">
+        <a href="#" class="btn btn-link">
+          ${iconTemplate({ icon: 'angle-right' })}
```

Comment:
Dit icoon is dus wél terecht. Idealiter zouden we dit eigenlijk met een icon after met `di()` mixin doen